### DISCUSSION TITLE
Drop tool documentation and examples

### DISCUSSION THREAD

**Discussion by ionutpredoaia** [eclipse-sirius/sirius-components#3469] [Q&A]
Hello,

In Sirius Web, what is the difference between the Drop Tool and the Drop Node Tool?

![siriusweb_droptool](https://github.com/eclipse-sirius/sirius-web/assets/7106155/3f3dc0bb-7272-4146-99f6-3cc7b3889754) 

Is there any documentation available on how and when they should be used? Are there any examples available of their usage? 

For instance, do any of them support dragging and dropping one model element from the diagram (not from the tree editor) into a container?

Thanks,
Ionut

**Comment by sbegaudeau** [comment]
Hi,

You can find an example of the Drop Tool [here](https://github.com/eclipse-sirius/sirius-web/blob/0b889ec075a94eaa655d2a528cf3ee61d6115f7c/packages/sirius-web/backend/sirius-web-papaya/src/main/java/org/eclipse/sirius/web/papaya/representations/componentdiagram/tools/ComponentDiagramDropToolProvider.java). Drop tools are used to drop an element from the explorer into the diagram. To move an element from one containing node to another, you should use Drop Node Tools.

A new Sirius Web documentation website is currently being created, it will be available soon.

Regards,

Stephane Begaudeau
